main/src/ca/uwaterloo/flix/language/phase/Instances.scala

```diff
@@ -169,4 +190,14 @@ object Instances extends Phase[TypedAst.Root, TypedAst.Root] {
       case ((), x) => f(x)
     }
   }
+
```

Comment:
I feel like checkEach is very close to `traverse`?

Reply:
Yes, the difference that it ignores the output, similar to `traverse_` in Haskell. Using traverse would give me a `List[Unit]` at the end.